editions/es-ES/tiddlers/es-ES/ControlPanel.multids

```diff
@@ -0,0 +1,141 @@
+title: $:/language/ControlPanel/
```

Comment:
I'm afraid there's another issue here: these tiddlers are actual Spanish translation of the core strings. They should be in `languages/es-ES`, not here in the `es-ES` edition.